### ISSUE TITLE
[Feature Request] - multiple workers with a single database (MongoDB or a single .session Folder)

### ISSUE THREAD

**Issue by HuangDaHui** [allburov/whatsapp-http-api#525]
This is the case with WEBJS or NOWEB, I don’t know how to avoid it.

**Comment by HuangDaHui** [comment]
Perhaps adding the MongoDB database name prefix configuration can solve this problem?

**Comment by devlikepro** [comment]
Yes, right now only way to use is to use different databases :( 
Or when you use a single one you need to set `WHATSAPP_RESTART_ALL_SESSIONS=False` so it doesn't run ALL sessions in ALL workers

We'll work on that so you can use signle database with multiple WAHA workers!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by HuangDaHui** [comment]
> Yes, right now only way to use is to use different databases :( Or when you use a single one you need to set `WHATSAPP_RESTART_ALL_SESSIONS=False` so it doesn't run ALL sessions in ALL workers
> 
> We'll work on that so you can use signle database with multiple WAHA workers!
> 
> [![patron:PRO](https://camo.githubusercontent.com/4f70c3a9ee2450c288faea5e1a1b7ebd0ffc1571450a59676b300ffb32b32537/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

OK, looking forward to your updates, thank you
